test/buffer/worker/expected-etcd-state/_config_001.txt

```diff
@@ -5,7 +5,7 @@ config/export/%%TEST_KBC_PROJECT_ID%%/my-receiver/my-export-1
   "projectId": %%TEST_KBC_PROJECT_ID%%,
   "receiverId": "my-receiver",
   "exportId": "my-export-1",
-  "name": "my-export-1",
+  "name": "My Export 1",
```

Comment:
V testoch som zmenil `name` aby nebolo rovnake ako `id`, tak to menim aj v tomto snapshote/ocakavanom stave.